src-built-in/adapters/indexedDBAdapter.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * IndexDB Storage Adapter.
+ * 
+ * We have a baseStorage model that provides some methods, such as `getCombinedKey`, which will return a nice key to 
+ * save our value under. Example: `Finsemble:defaultUser:finsemble:activeWorkspace`. That key would hold the value of 
+ * our activeWorkspace.
+ */
+import finsemble from "@chartiq/finsemble";
+
+const SCHEMA_VERSION = 1;
+
+const BaseStorage = finsemble.models.baseStorage;
+const Logger = finsemble.Clients.Logger;
+
+// Because calls to this storage adapter will likely come from many different windows, we will log successes and 
+// failures in the central logger.
+Logger.start();
+
+const IndexedDBAdapter = function () {
+	BaseStorage.call(this, arguments);
```

Comment:
@thorsent I tried to change this to an ES6 class, but I got error when this method was called as:
```super(arguments);```
If you rework this interface, we should keep an eye on being able to support ES6 classes.